modules/lib/src/main/resources/assets/js/v6/features/layout/AppShell/AppBar.tsx

```diff
@@ -1,13 +1,24 @@
 import {LegacyElement} from '@enonic/lib-admin-ui/ui2/LegacyElement';
 import {Button, IconButton} from '@enonic/ui';
-import {ArrowLeftRight, BellIcon, LayoutGrid} from 'lucide-react';
+import {ArrowLeftRight, BellDotIcon, BellIcon, LayoutGrid} from 'lucide-react';
 import {ReactElement} from 'react';
 import {ProjectSelectionDialog} from '../../../../app/dialog/ProjectSelectionDialog';
 import {Store} from '@enonic/lib-admin-ui/store/Store';
 import {ShowIssuesDialogEvent} from '../../../../app/browse/ShowIssuesDialogEvent';
 import {useI18n} from '../../../../app/ui2/hooks/useI18n';
+import {$issuesStats} from '../../store/issuesStats.store';
+import {useStore} from '@nanostores/preact';
 
 const AppBar = (): ReactElement => {
+    const {stats} = useStore($issuesStats);
+
+    const issuesStatsLabel =
+        stats?.openAssignedToMe > 0
+            ? useI18n('field.assignedToMeCount', String(stats.openAssignedToMe))
+            : stats?.open > 0
+              ? useI18n('field.openIssuesCount', String(stats.open))
+              : useI18n('field.noOpenIssues');
```

Comment:
Just avoid nested ternaries. It can either be an `if-else-return` here or in separate function. What I would actually do, I'd move it to a function before AppBar component and use if-ese ti return string keys, e.g.:
```ts
type LabelKeys = [`field.${string}`, ...string[]];

function createIssuesLabelKeys(stats: Stats | undefined): LabelKeys {
  if (stats?.openAssignedToMe > 0) {
    return ['field.assignedToMeCount', String(stats.openAssignedToMe)];
  }
  // ...
}

// ...
const issuesLabel = useI18n(...createIssuesLabelKey(stats));
```